frontend/src/components/RoomsPage/RoomForm/RoomForm.js

```diff
@@ -67,11 +68,11 @@ const RoomForm = (props) => {
                     label={t(TYPE_LABEL)}
                     validate={[required]}
                 >
-                    <option className="room-select-item" />
+                    <MenuItem value="" className="hidden" disabled />
```

Comment:
Why do we need this strange item?

Reply:
it's from material. I use it for style options in select and add effect of hover.